### PR TITLE
allow specifying publisher, offer, and sku for windows images

### PR DESCRIPTION
**Pull request by dmitsh**
Allow specifying publisher, offer, and sku for windows images

### CODE REVIEW

**Comment by ionutbalutoiu** [comment]
Looks good to me. Useful for the upstream [DC/OS Windows CI](https://mesos-jenkins.westus.cloudapp.azure.com/) using `acs-engine`.

File: parts/windowsparams.t [hunk 1]
```diff
@@ -23,4 +23,25 @@
...
+    },
+    "agentWindowsOffer": {
+      "defaultValue": "WindowsServer",
+      "metadata": {
+        "description": "The offer of windows image for the agent virtual machines."
+      },
+      "type": "string"
+    },
+    "agentWindowsSku": {
+      "defaultValue": "2016-Datacenter-with-Containers",
```
Comment (JiangtianLi) [inline parts/windowsparams.t]: Is this default value shared by k8s?

Comment (dmitsh) [reply parts/windowsparams.t]: yes

Comment (JiangtianLi) [reply parts/windowsparams.t]: k8s uses a different default "agentWindowsSku": "Datacenter-Core-1709-with-Containers-smalldisk".

**Pull request merged**